src/desktop/apps/article/routes.js

```diff
@@ -119,9 +116,19 @@ export async function index(req, res, next) {
         '../../../components/main_layout/templates/react_blank_index.jade'
     }
 
-    const isMobile = res.locals.sd.IS_MOBILE
+    const { ARTICLE_TOOLTIPS, CURRENT_USER, IS_MOBILE } = res.locals.sd
+
+    const isMobile = IS_MOBILE
     const jsonLD = stringifyJSONForWeb(articleModel.toJSONLD())
 
+    // Email signup
+    const isLoggedIn = typeof CURRENT_USER !== 'undefined'
+
+    // Tooltips a/b/c test
+    const isAdmin = isLoggedIn && CURRENT_USER.type === 'Admin'
+    const showTooltips = ARTICLE_TOOLTIPS !== 'control' && isAdmin
+    const showToolTipMarketData = ARTICLE_TOOLTIPS === 'market'
```

Comment:
should be ok since `showTooltips` will be false but we're not checking if `user is Admin` here.